packages/headless/src/features/pagination/pagination-slice.ts

```diff
@@ -105,11 +118,20 @@ export const paginationReducer = createReducer(
       .addCase(toggleSelectNumericFacetValue, (state) => {
         handlePaginationReset(state);
       })
+      .addCase(toggleSingleSelectDateFacetValue, (state) => {
```

Comment:
For these kind of repetition (where we need to do the same thing for multiple action), would there be a way to leverage `addMatcher()` ?

Reply:
If we leverage addMatcher, we open a pandora's box, where we'll try to match actions based on their type (string value) or content. Opens up a lot of possibilities for bugs.

Here we leverage the fact that we use typescript, and we import the exact actions, this way it's easier to maintain.